src/pages/login.tsx

```diff
@@ -0,0 +1,28 @@
+import profilePicture from '../images/demo-user.png';
+import Image from 'next/image';
+
+type LoginProps = {
+  logIn: (id: number) => void
+}
+const Login = ({logIn}: LoginProps) => {
+
+  return (
+    <section className='mt-10 flex flex-col items-center'>
+      <h1 className='text-4xl mt-10 celestial-cursive text-mellow-yellow text-center'>Celestial Cycle</h1> 
+      <button className="mt-10 relative group" onClick={() => logIn(1)}>
+        <div className="w-60 h-60 relative overflow-hidden">
+          <Image className='h-full w-full object-cover rounded-full' src={profilePicture} alt='drawing of woman with tattoo of lotus flower on her back'/>     
```

Comment:
Awesome work, i was confused about next's images when i first tried using them. I imagine it's still giving you a couple console errors because you have to specify the width and height as props for some reason? looks good tho!